# Define the goal
string filename
bool if_double
uint32 version
uint32 samp_f
uint32 tar_f
geometry_msgs/Point guess
---
# Define the result
float64 hz_angle
uint32 total_count
uint32 success_count
uint32 valid_count
---
# Define a feedback message
uint32 total_count
uint32 process_count
uint32 success_count
